Tests/SentryTests/Integrations/Performance/FramesTracking/SentryFramesTrackerTests.swift

```diff
@@ -710,6 +710,156 @@ class SentryFramesTrackerTests: XCTestCase {
         XCTAssertEqual(sut.isRunning, true)
     }
     
+    func testUnpause_ResetsPreviousFrameTimestamp_ToAvoidFalseAppHangDetection() throws {
```

Comment:
`l`: The frames tracker doesn't know the app hang detection exists. We validate that we don't record wrong frame statistics. The test names should reflect that.